src/cli/commandModules/copy.ts

```diff
@@ -17,7 +17,26 @@ const commandModule: CommandModule<
     return withIgnoreOption(withStateOption(withDestinationOption(yargs)));
   },
   handler: async (args: Arguments<CopyArgs>) => {
-    const errors = await copy(args);
+    const { errors, filesCopied } = await copy(args);
+    filesCopied.forEach(({ fromPath, toPath, type, errorMessage }) => {
```

Comment:
nit: Might be nice to break this into a separate function/interface like `report()` down the line - this adds some heft that isn't clearly "format and report the output" until you read through it